math/binary/abs.go

```diff
@@ -0,0 +1,12 @@
+package binary
+
+// ABS - this function return absolute value use binary operation
+// Principle of operation:
+// 1) Get the mask by right shift by the bit width of the variable (int64, int32, int16, int8)
```

Comment:
Since the method accepts an `int`, supporting other lengths is not required in my opinion. It will always be an `int`.

Reply:
> Since the method accepts an `int`, supporting other lengths is not required in my opinion. It will always be an `int`.

this is added only to describe the algorithm, to understand that the shift must be a multiple of the digit of your integer

Reply:
In my opinion, it can be adequately described in the comments, and a separate variable is redundant.

Reply:
done